projects/bounding_objects/protos/PipeBoundingObject.proto

```diff
@@ -14,7 +13,7 @@ PROTO PipeBoundingObject [
 {
   %{
     local wbrandom = require('wbrandom')
-
+    wbrandom.seed(context.id)
```

Comment:
I believe this is not needed and it's better to keep this proto deterministic.
The randomness here is used to make the contact points not on the same plane and hence improve the performance.
Using a fixed seed is perfectly fine.
```suggestion
```